install/homepage-install.sh

```diff
@@ -18,16 +18,16 @@ $STD apt-get install -y curl
 $STD apt-get install -y sudo
 $STD apt-get install -y mc
 $STD apt-get install -y git
+$STD apt-get install -y make
```

Comment:
This script had a few other packages mixed in with the node.js install, I moved them up here